targets/sim/base.py

```diff
@@ -32,11 +29,12 @@ def __init__(self, platform, **kwargs):
             kwargs['integrated_rom_size']=0x8000
         if 'integrated_sram_size' not in kwargs:
             kwargs['integrated_sram_size']=0x8000
-        if 'firmware_ram_size' not in kwargs:
-            kwargs['firmware_ram_size']=0x10000
-        if 'firmware_filename' not in kwargs:
-            kwargs['firmware_filename'] = "build/sim_{}_{}/software/firmware/firmware.fbi".format(
-                self.__class__.__name__.lower()[:-3], kwargs.get('cpu_type', 'lm32'))
+
+        firmware_ram_size=0x20000
+        firmware_filename="{}/build/sim_{}_{}/software/firmware/firmware.fbi".format(
```

Comment:
There should be a better way to get the firmware name here.  But I can't figure out how to pass it in `make.py` yet.